milestone2.ts

```diff
@@ -83,32 +83,31 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
-
+let findAverage: GetNumber = (data: Volunteer[]) => data.reduce((total, next) => total + next.age, 0) / data.length;
 //console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data: Volunteer[]) => data.findIndex((x) => x.city == "SF");
 
 //console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the people from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data: Volunteer[], n: Number) => data.filter(x=> (x.state == "CA") && (x.age > n)); // Code here
```

Comment:
FYI the input types are already declared in GetVolunteers, so you don't need to type them out again in the input for the function, but there's also no harm in doing so